src/webserver.cc

```diff
@@ -82,12 +84,17 @@ void Webserver::processConnection(int threadIndex, tcp::socket& socket) {
         }
     }
     std::string response = processRawRequest(req);
+    if (response == "") {
+        std::cout << "Error processing request\n";
```

Comment:
We can't really convert this into a separate function call, since we return right after. So we'd have to do an if and then return anyway, not saving any code space but making it harder to understand.